packages/react-server/src/ReactFizzAsyncDispatcher.js

```diff
@@ -16,8 +16,13 @@ function getCacheForType<T>(resourceType: () => T): T {
   throw new Error('Not implemented.');
 }
 
+function cacheSignal(): null | AbortSignal {
+  throw new Error('Not implemented.');
```

Comment:
Wait, wasn't that supposed to be a noop, same as for Fiber?

Reply:
It's a noop in the ReactClient layer for both of them. This never gets called.